Eatery/Controllers/Eateries/CollegetownEateriesMenuViewController.swift

```diff
@@ -6,80 +6,61 @@
 //  Copyright © 2019 CUAppDev. All rights reserved.
 //
 
+import Hero
 import Crashlytics
 import MapKit
 import UIKit
 
 class CollegetownEateriesMenuViewController: UIViewController, UIScrollViewDelegate, MKMapViewDelegate, CLLocationManagerDelegate {
     
     var eatery: CollegetownEatery
-    var delegate: MenuButtonsDelegate?
-    
+    weak var delegate: CampusMenuButtonsDelegate?
+
+    var navigationTitleLabel: UILabel!
     var outerScrollView: UIScrollView!
-    var backButton: UIButton!
-    var ctownMenuHeaderView: CollegetownMenuHeaderView!
+    var menuHeaderView: CollegetownMenuHeaderView!
     var informativeViews: [UIView]!
 
     var mapView: MKMapView
     let eateryAnnotation = MKPointAnnotation()
-    var locationManager: CLLocationManager!
-    var userLocation: CLLocation {
-        return locationManager?.location ?? .olinLibrary
-    }
+    let userLocation: CLLocation
 
     var defaultCoordinate: CLLocationCoordinate2D!
     var midpointCoordinate: CLLocationCoordinate2D!
     var delta: (lat: Double, lon: Double)!
     var maxDelta = (lat: 2/69.172, lon: 2/51.2738554594)
 
-    init(eatery: CollegetownEatery, delegate: MenuButtonsDelegate?){
+    init(eatery: CollegetownEatery, delegate: CampusMenuButtonsDelegate?, userLocation userLoc: CLLocation? = nil) {
         self.eatery = eatery
         self.delegate = delegate
         self.mapView = MKMapView()
+        self.userLocation = userLoc ?? .olinLibrary
+
         super.init(nibName: nil, bundle: nil)
         
         mapView.delegate = self
-        
-        locationManager = CLLocationManager()
-        locationManager.delegate = self
-        locationManager.desiredAccuracy = kCLLocationAccuracyBest
 
         defaultCoordinate = eatery.location.coordinate
 
-        midpointCoordinate = CLLocationCoordinate2D.midpoint(between: userLocation.coordinate,
-                                                             and: eatery.location.coordinate)
+        midpointCoordinate = .midpoint(between: userLocation.coordinate,
+                                       and: eatery.location.coordinate)
 
         delta = CLLocationCoordinate2D.deltaLatLon(between: userLocation.coordinate,
```

Comment:
I noticed how we can have just .midpoint instead of CLLocationCoordinate2D.midpoint (same for .olinLibrary), yet we cannot do the same for .deltaLatLon because we did not extend the type (lat: Double, lon: Double). Does it make sense to have deltaLon within the CLLocationCoordinate2D extension then?